config/settings/base.py

```diff
@@ -308,6 +308,14 @@
 SOCIALACCOUNT_ADAPTER = "commcare_connect.users.adapters.SocialAccountAdapter"
 # https://django-allauth.readthedocs.io/en/latest/forms.html
 SOCIALACCOUNT_FORMS = {"signup": "commcare_connect.users.forms.UserSocialSignupForm"}
+SOCIALACCOUNT_PROVIDERS = {
+    "commcarehq": {
+        "APP": {
+            "client_id": env("COMMCAREHQ_OAUTH_CLIENT_ID"),
+            "secret": env("COMMCAREHQ_OAUTH_SECRET"),
+        }
+    }
+}
```

Comment:
I know I asked you to put this in but I've since discovered that if we want to store the user's token (`SOCIALACCOUNT_STORE_TOKENS`) we do actually need to put the config in the DB and can't use this settings config.

Sorry for the back and forth.